presto-product-tests/src/main/java/com/facebook/presto/tests/convention/TestShowPartitions.java

```diff
@@ -56,17 +56,14 @@ public Requirement getRequirements(Configuration configuration)
 
     private static TableDefinition partitionedTableDefinition()
     {
-        StringBuilder createTableDdl = new StringBuilder();
-        createTableDdl.append("CREATE EXTERNAL TABLE %NAME%(");
-        createTableDdl.append("   col INT");
-        createTableDdl.append(") ");
-        createTableDdl.append("PARTITIONED BY (part_col INT) ");
-        createTableDdl.append(" STORED AS ORC");
+        String createTableDdl = "CREATE EXTERNAL TABLE %NAME%(col INT) " +
+                "PARTITIONED BY (part_col INT) " +
+                "STORED AS ORC";
```

Comment:
You can also fixup ./src/main/java/com/facebook/presto/tests/hive/TestTablePartitioningSelect.java, it uses `StringBuilder` in similar way

Reply:
separate PR